templates/admin/polls/index.html

```diff
@@ -0,0 +1,14 @@
+<% include adminheader.ejs %>
+<script>
+function createPoll() {
+	socket.emit('createPoll')
+	window.location.href = "/admin/poll/create"
+}
+</script>
+<div class="top"><h1>Polls</h1></div>
+<div class="search"><input id="searchBar" type="text"></div>
+<div class="main">
+<button onclick="">Create poll</button>
```

Comment:
I wouldn't advice the usage of `onclick` on html tags. It's a bad practice. The best way to go around this is by finding the DOM node and hooking an eventListener to it. Something like this:

```html
<!-- Your template file -->
<div class="main">
  <button class="js-create-poll-button">Create poll</button>
</div>
```

```javascript
// Your js file
(function() {
  // First find the button you want to add the event listener to
  var isAttached = false;
  var button = document.querySelector('.js-create-poll-button');

  function createPoll() {
    socket.emit('createPoll');
  }

  // If the button does not exist on the page, stop the script here
  if (!button) return;
  
  // If the button already is listening to the click event, then
  // don't attach an event listener to it
  if (isAttached) return;

  button.addEventListener('click', createPoll);
  isAttached = true;
})();
```